Sources/TensorFlow/Layers/Core.swift

```diff
@@ -12,17 +12,29 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-public extension Tensor where Scalar: TensorFlowFloatingPoint {
+fileprivate extension Tensor where Scalar: TensorFlowFloatingPoint {
     /// Computes dropout given a probability.
     @differentiable(wrt: self where Scalar: Differentiable)
-    func droppingOut(probability: Double) -> Tensor {
+    fileprivate func _droppingOut(probability: Double) -> Tensor {
```

Comment:
Since the `extension` has been marked as `fileprivate`, access modifiers on individual method declarations can be dropped.
```suggestion
    func _droppingOut(probability: Double) -> Tensor {
```